target/linux/ipq40xx/config-5.4

```diff
@@ -299,7 +233,7 @@ CONFIG_MDIO_BITBANG=y
 CONFIG_MDIO_BUS=y
 CONFIG_MDIO_DEVICE=y
 CONFIG_MDIO_GPIO=y
-CONFIG_MDIO_IPQ40XX=y
+# CONFIG_MDIO_IPQ4019 is not set
```

Comment:
And probably that change should not be in this patch?

Reply:
Can I simply include it in the MDIO backport patch?
To me, it makes no sense as a standalone change

Reply:
Yes, that's what I meant.

Reply:
Ok, I have already pushed the updated version with that.